insolar/insolar
# GH actions tag event

**- What I did**
Added release workflow to the repo
**- How I did it**
I've added Dockerfile and GH actions workflow pipeline which is being triggered by tag event. 
Every tag event will trigger build, after that docker image will be created and pushed to docker registry and new GH release will be added to releases page, w/ binaries produced by build stage.
**- Requirements**
Those secrets should be added to the repo for workflow to work properly:
```
REGISTRY_URL
REGISTRY_USERNAME
REGISTRY_PASSWORD
```
**- How to verify it**
You can tag any commit with any tag name like that:
```
$ git tag -a 'v1.0.0' -m 'v1.0.0' 
$ git push origin v1.0.0
```
After steps above you should be able to download artifacts from release page and pull image from registry:
```
docker pull insolar/insolar:v1.0.0
```

**- Description for the changelog**
GH Actions release pipeline
